Formula/ucloud.rb

```diff
@@ -0,0 +1,22 @@
+class Ucloud < Formula
+  desc "The official tool to managment your ucloud services"
+  homepage "https://www.ucloud.cn"
+  url "https://ucloud-sdk.dl.ufileos.com/ucloud-cli-0.1.2.tar.gz"
+  sha256 "f41aaa6bf8063b3ccd0c5e41d3dc90f46f9fb45066513da8e306456f1f332071"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/ucloud/ucloud-cli"
+    dir.install buildpath.children
+    cd dir do
+      system "go", "build", "-o", bin/"ucloud"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "#{bin}/ucloud", "config", "ls"
```

Comment:
Can we match (part of) the output of the test? `assert_match` or `assert_equal`

Reply:
>```
>test do
>   system "#{bin}/kube-aws"
>    system "#{bin}/kube-aws", "init", "--cluster-name", "test-cluster",
>           "--external-dns-name", "dns", "--region", "us-west-1",
>           "--availability-zone", "zone", "--key-name", "key",
>           "--kms-key-arn", "arn", "--no-record-set",
>           "--s3-uri", "s3://examplebucket/mydir"
>    cluster_yaml = (testpath/"cluster.yaml").read
>    assert_match "clusterName: test-cluster", cluster_yaml
>    assert_match "dnsName: dns", cluster_yaml
>    assert_match "region: us-west-1", cluster_yaml
>    assert_match "availabilityZone: zone", cluster_yaml
>    assert_match "keyName: key", cluster_yaml
>    assert_match "kmsKeyArn: \"arn\"", cluster_yaml
>    installed_version = shell_output("#{bin}/kube-aws version 2>&1")
>    assert_match "kube-aws version v#{version}", installed_version
> end
```
I found an example.   I will have a try.